docs/changelog/changelog_en.md

```diff
@@ -1,38 +1,36 @@
 # Changelog plentyshopPWA
 
-## v1.4.1 (2024-xx-xx) <a href="https://github.com/plentymarkets/plentyshop-pwa/compare/v1.4.0...v1.4.1" target="_blank" rel="noopener">
+## v1.4.1 (2024-06-04) <a href="https://github.com/plentymarkets/plentyshop-pwa/compare/v1.4.0...v1.4.1" target="_blank" rel="noopener">
 
 ### New
 
 - Add custom SVG icon support
 - Lazy-load images
-- a guest login page before going to checkout
-- Width and height attributes to images on category and product view
+- A guest login page before going to checkout
 - Selection for the saved addresses at checkout
+- Width and height attributes to images on category and product view
 - Product reviews on single item pages
-- Product renaming in the documentation files
 - Added quick checkout functionality.
 - Added quantity to quick checkout product name
```

Comment:
This is not relevant because its part of the  “Item added to shopping cart” dialog feature